app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -223,6 +223,19 @@ def self.event_monitor_class
     self::EventCatcher
   end
 
+  def host_quick_stats(host)
+    qs = {}
+    with_provider_connection(:version => 4) do |connection|
+      l = connection.system_service.hosts_service.host_service(host.uid_ems)
```

Comment:
I think it's likely that the meaning of `l` is going to get lost.  Can you change this to `stats_list`, or something similar?

Also, below, the `l` almost looks like the numeral `1`, and I was hugely confused as to what `1.detect` meant. :)